docs/guide/index.md

```diff
@@ -115,7 +115,8 @@ Running `vite` starts the dev server using the current working directory as root
 
 In a project where Vite is installed, you can use the `vite` binary in your npm scripts, or run it directly with `npx vite`. Here is the default npm scripts in a scaffolded Vite project:
 
-```jsonc
+<!-- prettier-ignore -->
```

Comment:
**question:** Why do we need this ignore?

Reply:
Prettier formatted to single quote. So, we need to ignore for this. See https://github.com/vitejs/vite/commit/60dab0372c83264f41fbd3eeb204abf00c2468c9